reposilite-site/data/guides/features/dashboard.md

```diff
@@ -3,36 +3,33 @@ id: dashboard
 title: Dashboard
 ---
 
-Reposilite exposes frontend that is mounted to `/#/` as SPA application in Vue 3.
-You can disable frontend in [shared configuration](/guide/settings#shared-configuration)
-if you just want to disable an easy way to query your repository or to provide custom frontend implementation using [static files](/guide/static-files).
+Reposilite exposes a frontend that is mounted to `/#/` as [SPA (_single-page application_)](https://en.wikipedia.org/wiki/Single-page_application) in [Vue 3](https://vuejs.org/). You can disable the frontend in [shared configuration](/guide/settings#shared-configuration) if you want to disable an easy way to query your repository or to provide a custom frontend implementation using [static files](/guide/static-files).
 
 #### Browser
-Browser displays content of currently selected directory, respectively to provided authentication credentials. For not logged user it lists only public files, for authenticated one it also displays files that user has access to.
+Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.
```

Comment:
```suggestion
Browser displays the contents of the currently selected directory in respect to the provided authentication credentials. For unauthenticated users, only public files will be listed. For authenticated users, all files that the user has access to will displayed.
```

I think these changes would help make this sentence clearer and more consistent.

Reply:
With your suggestion there's a change of time in the paragraph: "Browser displays" – "will be listed" – "will [be] displayed". 

Change of "not logged in" to "unauthenticated" is a good idea.